chunkator/__init__.py

```diff
@@ -11,9 +11,12 @@ def chunkator(queryset, chunk_size):
     length. We're saving at least a ``count()`` query on QuerySets, or a
     CPU-and-RAM-consuming ``len(queryset)`` query.
     """
-    pk = 0
+    pk = None
     while True:
-        page = queryset.filter(pk__gt=pk).order_by('pk')[:chunk_size]
+        queryset = queryset.order_by('pk')
```

Comment:
@brunobord I'm definately not agree with you about miss record when the export is running, what do y ou want to do ? Export all record that exists the end of the export, or all records that exists when you start your export ? You'll always miss the record just created the 1ms after your export finished.
And sure tke key pagination works only on unique ordered keys.